migrate.hcl

```diff
@@ -0,0 +1,21 @@
+description = "Database migrations written in Go. Migrate reads migrations from sources and applies them in correct order to a database."
+test = "migrate --version"
+binaries = ["migrate"]
+
+version "v4.15.2" {
```

Comment:
The version number can't include `v`. You'll have to insert the v below into the URLs.